internal/zinc-core/src/main/java/xsbti/compile/ScalaJsFiles.java

```diff
@@ -0,0 +1,22 @@
+package xsbti.compile;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Can be added to `IncOptions.auxiliaryClassFiles` so that the sjsir files
+ * produced by the Scala.js compiler are managed by the ClassFileManager.
+ */
+public class ScalaJsFiles implements AuxiliaryClassFiles {
```

Comment:
I don't know what's the convention of this repo for acronyms in class names, but in Scala.js we typically `ScalaJS`, for example `ScalaJSPlugin`. Under that convention, it would be nice for this to be called `ScalaJSFiles`. However, that may conflict with the conventions of this repo, which should prevail.